homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -112,7 +112,7 @@ def device_state_attributes(self):
 
         return {}
 
-    def get_state(self, after_shkia_date, after_tzais_date):
+    def get_state(self, date, after_shkia_date, after_tzais_date):
```

Comment:
I think, `after_midnight_date` would be clearer, or at least update the docstring explaining what the three dates are.

Reply:
What do you think about calling it "daytime_date"? That would be unambiguous and fit the pattern of the other two.

Reply:
Go for it.